src/db/config.ts

```diff
@@ -32,7 +32,7 @@ if (environment == 'production') {
   var dbName: string;
   var dbUser: string;
   var dbPassword: string;
-  
+
```

Comment:
We should probably add some eslint/prettier integrations to prevent noisy diiffs like this